code/data_structures/linked_list/circular_linked_list/isCircular.py

```diff
@@ -0,0 +1,16 @@
+
+# uses try block so dont have to check if reached the end, provides faster runtime. credit-leetcode
+def is_circular(self, head):
+	'''
+	function checks if a link list has a cycle or not
+	'''
+	try:
+		slow = head
+		fast = head.next
+
+		while slow is not fast:
+			slow = slow.next
+			fast = fast.next.next
+		return true
```

Comment:
In Python, true and false are capitalized.

Reply:
teehee did u test it

Reply:
@deesims A quick google search, along with documentation, confirms this. Here is also an [example](https://repl.it/Ngrh/0) in Python 3. Here is one in [Python 2](https://repl.it/Ngrp/0).

Reply:
yeah i was asking @ayush94 if he tested it, clearly 'true' doesnt parse in python compiler